vita3k/renderer/src/texture_yuv.cpp

```diff
@@ -8,10 +8,35 @@ extern "C" {
 #include <libswscale/swscale.h>
 }
 
+
+
 namespace renderer::texture {
+
+    static SwsContext *renderSwsContext = nullptr;
+    static size_t res[2] = {0,0};
+SwsContext *getSwsContext(size_t width, size_t height) {
```

Comment:
```suggestion
SwsContext *get_sws_context_(size_t width, size_t height) {
```
this is our code style everywhere